app/src/main/kotlin/cz/covid19cz/erouska/DI.kt

```diff
@@ -86,6 +87,7 @@ val appModule = module {
     single { WakeLockManager(androidContext().getSystemService()) }
     single { androidContext().getSystemService<PowerManager>() }
     single { androidContext().getSystemService<BluetoothManager>() }
+    single { Markdown() }
```

Comment:
Bylo by hezký tam teda injectnout ten context když už DI používáme

Reply:
Jo to máš recht. Udělám